package.json

```diff
@@ -11,21 +11,20 @@
     "dependencies": {
         "artedi": "1.1.0",
         "assert-plus": "1.0.0",
-        "bignum": "0.6.0",
+        "bignum": "0.13.0",
         "bunyan": "0.22.1",
         "clone": "0.1.9",
-        "dtrace-provider": "0.2.8",
-        "fash": "2.5.0",
+        "dtrace-provider": "0.8.8",
+        "fash": "git+https://github.com/joyent/node-fash.git#MANTA-4720",
         "fast": "2.7.0",
         "jsprim": "2.0.0",
         "kang": "1.2.0",
-        "ldapjs": "0.6.3",
         "imgapi-cli": "git+https://github.com/joyent/sdc-imgapi-cli.git#v2.4.1",
         "boray": "git+https://github.com/joyent/node-buckets-mdapi.git#v0.1.0",
-        "node-uuid": "1.4.0",
         "posix-getopt": "^1.0.0",
         "restify": "6.3.1",
         "strsplit": "1.0.0",
+        "uuid": "3.3.3",
         "vasync": "^2.2.0",
         "verror": "1.9.0",
         "xtend": "2.0.6"
```

Comment:
'deep-equal' could be dropped if you like. It isn't used (drop the unused import in test/helper.js).  It isn't used in electric-moray.git from whence this came either. I don't think it ever was.

Reply:
interesting, will do.